pkg/resolver/searchComplete.go

```diff
@@ -48,6 +48,18 @@ func (s *SearchCompleteResult) searchCompleteQuery(ctx context.Context) {
 	var selectDs *goqu.SelectDataset
 
 	//FROM CLAUSE
+	// if s.input.Keywords != nil && len(s.input.Keywords) > 0 {
```

Comment:
Why is this block commented? Remove if not needed or add explanation in the code if it's something that we'll need later.